quaint/src/tests/query.rs

```diff
@@ -3645,7 +3645,7 @@ async fn traceparent_is_stripped_from_the_log(api: &mut dyn TestApi) -> crate::R
         .query_raw("SELECT 1 /* traceparent=1 */", &[])
         .await?
         .into_single()?;
-    let expected = r#"db.statement=SELECT 1 otel.kind="client""#.to_owned();
+    let expected = r#"db.query.text=SELECT 1 otel.kind="client""#.to_owned();
```

Comment:
Reviewer's note: this aligns with the OpenTelemetry spec conventions: https://opentelemetry.io/docs/specs/semconv/database/database-spans/#sanitization-of-dbquerytext